gcsfs/tests/test_fuse.py

```diff
@@ -29,6 +29,7 @@ def test_fuse(token_restore):
         with open(os.path.join(mountpath, 'hello'), 'w') as f:
             # NB this is in TEXT mode
             f.write('hello')
+        time.sleep(1)
```

Comment:
Typically i would do something like the following:

```python
start = time()
while 'hello' not in os.listdir(mountpath):
    sleep(0.1)
    assert time() < start + 5
```

However this may not work well with VCR.  In general I'm surprised that VCR wouldn't result in entirely equivalent results on travis-ci

Reply:
A good idea if this works, but I don't know the basic reason it's different. Maybe fuse on travis works somehow differently, laggy or with different caching.

Reply:
...but it still doesn't work :(

Reply:
Note that test does pass in docker linux. Should that be good enough for merger?